andrey/NewHW1/NewTask1.js

```diff
@@ -0,0 +1,46 @@
+
+function bookManger() {
+    let bookname = window.prompt ("Please, enter a book name: ");
+    let bookauthor = window.prompt ("Pleas, enter a book author: ");
+    let bookyear = window.prompt ("Please enter a book publication year: ");
+    let pagecount = window.prompt ("Please enter a book pages count: ");
+
+    window.confirm("Would you like to add " + bookname + " to the library ?");
+    if (window.confirm = true){
+        window.alert(bookname + " by "  + bookauthor + " successfully saved to the library!")
+    }
+    else {
+
+        }
+
+    return bookLibarary = {

```

Comment:
no declaration of **bookLibarary** found, furthermore, you don't need to declare any variables, let's just return an object